inc/classes/class-google-auth.php

```diff
@@ -345,7 +345,7 @@ public function authenticate_user( $user = null ) {
 
 		$user_info = $this->_get_user_from_token( $token );
 
-		if ( empty( $user_info ) || ! is_array( $user_info ) || empty( $user_info['user_email'] ) || ! is_email( $user_info['user_email'] ) ) {
```

Comment:
Whats the reason of this change? @sagarnasit

Reply:
I found this during code review. `empty( $user_info )` is being an extra check here. Other conditions after that are taking care of all the required checks.

Reply:
Got it. makes sense